realsense2_camera/test/live_camera/rosci.py

```diff
@@ -0,0 +1,147 @@
+# Copyright 2023 Intel Corporation. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys, os, subprocess, re, platform, getopt, time
+sys.path.append( os.path.join( os.environ['WORKSPACE'], 'lrs/unit-tests/py' ))
+from rspy import log, file, repo, libci
+start_time = time.time()
+current_dir = os.path.dirname( os.path.abspath( __file__ ) )
+print(f'{current_dir}')
+root = os.path.dirname( os.path.dirname( os.path.dirname( os.path.dirname( os.path.dirname( os.path.abspath( __file__ ))))))
```

Comment:
This looks very weird, what's going on here?

Reply:
restructured in commit # 5